### DISCUSSION TITLE
Testes con heurística. Públicos o no?

### DISCUSSION THREAD

**Discussion by marcgc21** [ruxailab/RUXAILAB#158] [Ideas]
Queremos abrir una discusión para hablar sobre la necesidad o no de crear un sistema para poder mostrar pruebas con heurísticas públicas. Nos gustaría oír la opinión de todos los interesados ​​para saber cómo evolucionar la herramienta.

Propongo intentar responder las siguientes preguntas:

- ¿Tiene sentido poder publicar los resultados de una evaluación heurística donde se podrían consultar los diferentes resultados de cada evaluador, así como sus comentarios e imágenes adjuntas?

Por mi lado lo veo interesante porque ayudaría con la reproducibilidad de los resultados ya que muchas veces en otros papers que se han realizado algun tipo de estudio de este tipo no he encontrado información sobre les comentarios, y de este modo tendríamos como una base de datos con información que actuaría como repositorio.

¿Y vosotros que pensáis?

Att,

M.

**Comment by tgranollers** [comment]
Hola,
yo creo que una de las opciones mejores del sistema que estamos creando es que pueda haber una comunicación-colaborción entre los evaluadores.
Pero, no debemos hacer un sistema de comunicación global, yo me centraría en que a nivell de heurística el evaluador pueda ver los comentarios de los otros evaluadores (a modo de los comentarios de un documento word con colaboración) ... lo que no tengo claro es si poner el nombre real del evaluador o un acrónimo para que no se identifique (y condicione)
